src/coordinator/block/column.go

```diff
@@ -62,6 +62,15 @@ func (c *columnBlock) SeriesMeta() []SeriesMeta {
 	return c.seriesMeta
 }
 
+// Steps returns the total steps
+func (c *columnBlock) Steps() int {
+	return len(c.columns)
+}
+
+func (c *columnBlock) Series() int {
```

Comment:
nit: missing comment

Reply:
probably better to just have them on the interface ? All implementations will have the exact same comments repeated